sweep/sweeper.go

```diff
@@ -252,12 +270,15 @@ func (s *UtxoSweeper) Stop() error {
 }
 
 // SweepInput sweeps inputs back into the wallet. The inputs will be batched and
-// swept after the batch time window ends.
+// swept after the batch time window ends. A custom fee preference can be
+// provided, otherwise the UtxoSweeper's default will be used.
```

Comment:
Why would we offer a default? I think it is good to make it a required parameter and force callers to think about it.

Reply:
Was thinking of the case where in the future we allow a flag that specifies the default fee preference of users. The flag doesn't exist yet, but don't think it'd hurt to keep this as is for now.

Reply:
It isn't an important detail, but there is an assumption in it that there will be a single default fee preference for all input sources. It may also be that we will assign different fee rates to different sources and then the defaults need to move to the callers.

Reply:
Done.